syntax = "proto3";

package NProto;

message TPutRequest {
    uint64 request_id = 1;
    string key = 2;
    uint64 offset = 3;
}

message TPutResponse {
}

message TGetRequest {
    uint64 request_id = 1;
    string key = 2;
}

message TGetResponse {
    uint64 offset = 1;
}
